include/capstone.h

```diff
@@ -27,19 +27,23 @@ extern "C" {
 #pragma warning(disable:4201)
 #pragma warning(disable:4100)
 #define CAPSTONE_API __cdecl
+#ifndef CAPSTONE_EXPORT
```

Comment:
The intention is to allow custom (build-system defined) export rules, e.g. some namespace prefix.